addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -283,6 +285,53 @@ var Wysiwyg = Widget.extend({
             mediaDialog.on('cancel', this, resolve);
         });
     },
+    changeSelectedTextColor(color, closeColorPicker = false) {
+        if(color === "") {
+            this.editor.execCommand('uncolorText');
+            $(window).trigger('click');
+            return;
+        }
+
+        if(this.colorPicker.colorNames.indexOf(color) !== -1) {
+           color = "var(--" + color + ")";
+        }
+        this.editor.execCommand('colorText', {color: color});
+
+        if(closeColorPicker) {
+            $(window).trigger('click');
```

Comment:
`ColorPicker.close()`?

Reply:
I also need to clean the colorPicker Container I use to host the widget and clean some variables